src/citation_vs_reviewscore.py

```diff
@@ -0,0 +1,15 @@
+import csv
+import os
+import sys
+
+src_path = os.getcwd()
+data_path = src_path.strip('src') + 'data/'
+input_file = 'acta_a_citations_review_scores_utf8.csv'
+file_path = data_path + input_file
+
+with open(file_path,mode='r',encoding='utf-8') as file:
+    data = list(csv.reader(file,delimiter=';'))
```

Comment:
I would leave data as an iterator, don't turn it into a list. Also,  maybe think of using DictReader so you get the rows back as dictionaries.  The dictionary keys are the column titles which I think would work for your csv.    For large cvs files we could also use pandas, but here I think what you are doing is better.

Reply:
Sounds as a good idea with the DictReader. We will implement this. At some point, I would like to briefly discuss when to use pandas and the possibilities offered by pandas.